docs/framework/get-started/the-net-framework-and-out-of-band-releases.md

```diff
@@ -47,8 +47,8 @@ OOB releases for core common language runtime (CLR) components are delivered thr
 
  To find both prerelease and stable packages, choose **Include Prerelease** in the NuGet Package Manager.
 
- If you want to be notified of stable package releases, subscribe to [the .NET Framework feed](https://nuget.org/api/v2/curated-feeds/dotnetframework/Packages/).
+ If you want to be notified of stable package releases, subscribe to [the .NET Framework feed](https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/Packages/).
```

Comment:
```suggestion
 If you want to be notified of stable package releases, subscribe to [the .NET Framework feed](https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/).
```

This is taken from [Microsoft .NET Framework NuGet Packages](https://devblogs.microsoft.com/dotnet/p/nugetpackages/) and should be the only Nuget feed.

See also [Nuget Feed Address](https://stackoverflow.com/questions/5578834/nuget-feed-address) on SO.

Reply:
We should delete the entire line. We no longer maintain the Microsoft and .NET feed which is why it no longer ships in the default feeds.

The replacement for them is [NuGet's package identity and trust features](https://blog.nuget.org/20170417/Package-identity-and-trust.html).